src/seedu/addressbook/data/tagging/Tagging.java

```diff
@@ -0,0 +1,63 @@
+package seedu.addressbook.data.tagging;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+public class Tagging {
+	final static String PREFIX_ADDED = "+ ";
+	final static String PREFIX_DELETED = "- ";
+	final static String PADDING = " ";
+	final Boolean isAdd;
+	final Person person;
+	final Tag tag;
+	
+	
+	public static ArrayList<Tagging> processTaggings(Person person, UniqueTagList tags){
+		ArrayList<Tagging> taggings = new ArrayList<>();
+		for(Tag tagToBeRemoved :  person.getTags()){
+			if(!tags.contains(tagToBeRemoved)){
+				Tagging tagging = new Tagging(false, person, tagToBeRemoved);
+				taggings.add(tagging);
+			}
+		}
+		for(Tag tag: tags){
+			if(!person.getTags().contains(tag)){
+				Tagging tagging = new Tagging(true, person, tag);
+				taggings.add(tagging);
+			}
+		}
+		return taggings;
+	}
+	
+	public Tagging(){
```

Comment:
This constructor is unnecessary here as it does not make much sense to instantiate a tagging object with no useful information at all. If this is not expected behavior, then try not to implement it.